test/unit/test_inference.py

```diff
@@ -311,7 +311,7 @@ def test_failed_make_input_from_valid_json_string(text, text_key, factors, facto
 @pytest.mark.parametrize("strings",
                          [
                              ["a b c"],
-                             ["a b c", "f1 f2 f3", "f3 f3 f3"]
+                             ["a b c", "f1 f2 f3", "f3 f3 f3"],
```

Comment:
probably not an issue, but adding the comma could be reverted.

Reply:
It was just because I remove the constraints manually. Fixing it.